packages/step-wizard/WizardStepTitle.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+const WizardStepTitle = ({ tag: Tag, className: classes, children }) => (
+  <Tag className={classNames(classes, 'stepwizard-title')}>{children}</Tag>
```

Comment:
Finish spreading the rest of the props on Tag.

Reply:
163fdf01c59f2a5b5015f765064b27c2d29e9e37